tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+reload(sys)
+
+# Command line arguments parsing
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-f1", "--mobile_api_file",
+    type=str, default="MOBILE_API.xml",
+    help='Enter mobile api file name! ("MOBILE_API.xml" by default)'
+)
+
+parser.add_argument(
+    "-f2", "--hmi_api_file",
```

Comment:
@AByzhynar f1 and f2 is rely confusing and easy to mismatch. Please either remove short args or give them more appropriate name